### PR TITLE
Advisor miscellaneous improvements

### PR DESCRIPTION
**Pull request by georgebisbas**
- Minor doc improvement
- Add a relative path to improve logging

### CODE REVIEW

File: benchmarks/user/advisor/README.md [hunk 1]
```diff
@@ -10,6 +10,9 @@ After the run has finished you should be able to plot a roofline with the result
 To export roofline data to JSON, use:
 * `python3 advisor_to_json.py --name Roofline --project <advisor-project-name>`
 
+To create a read-only snapshot for use with Intel Advisor GUI, use:
+* `advixe-cl --snapshot --project-dir=<advisor-project-name> pack -- /<new-snapshot-name>`
```
Comment (FabioLuporini) [inline benchmarks/user/advisor/README.md:14]: what's a snapshot?

File: benchmarks/user/advisor/README.md [hunk 2]
```diff
@@ -25,7 +28,7 @@ Limitations:
 * Running the `tripcounts` analysis takes a lot, despite starting in paused
   mode. This analysis, together with the `survey` analysis, is necessary to
   generate a roofline. Both are run by `run_advisor.py`.
-* Requires python3, untested in earlier versions of python
+* Requires python3, untested in earlier versions of python and conda environments
```
Comment (FabioLuporini) [inline benchmarks/user/advisor/README.md:31]: full stop (here and line below)

**Review by FabioLuporini — APPROVED** [review]
Basically uncontroversial

File: benchmarks/user/advisor/README.md (hunk 1 above)
Comment (georgebisbas) [reply benchmarks/user/advisor/README.md:14]: Snapshot is this small in size file used to save the results and see them efficiently in GUI.
https://software.intel.com/content/www/us/en/develop/documentation/advisor-user-guide/top/results/creating-a-read-only-result-snapshot.html

**Pull request merged**